lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/DriverStatusHTTPHandler.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.bridge.client;
```

Comment:
Should this class be in common.driver.client? It is not for bridge, right?

Reply:
I write it exclusively for the bridge. I wouldn't want to promote it to the `common` package just yet. We can do that if / when we redesign the client. The Java client already has a (different) way to know when the Driver is done and doesn't need this code.

Reply:
When you say bridge, you mean for .Net client to use, right? Our bridge code mostly is to handle the calls between .Net and Java. Although this HttpHandler is mainly used for .Net, as a web service, it can be called anywhere. From that perspective, it is not only for .Net and also it doesn't delegate any calls between Java and .Net.  That is what I mean it doesn't belong to bridge.

Reply:
I don't want to expose this as a new API to Java devs. They have a better way to do this already. I'm not set on where in the Java-for-.NET code to put it, though. Is there a better place for it?